bootstrap.sh

```diff
@@ -12,7 +12,7 @@ CURRENT_DIR=$(pwd)
 # default to clang
 CXX=${CXX:-clang++}
 TARGET=${TARGET:-Release}
-OSRM_RELEASE=${OSRM_RELEASE:-"develop"}
+OSRM_RELEASE=${OSRM_RELEASE:-"c++14"}
```

Comment:
You need to also update `OSRM_RELEASE` in `.travis.yml`. This is only used for local builds!